### PR TITLE
feat: update sb version

### PR DESCRIPTION
**Pull request by grorge123**
Update Rust SDK to use [new sb plugin](https://github.com/WasmEdge/WasmEdge/pull/3709).

### CODE REVIEW

File: rust/src/lib.rs [hunk 1]
```diff
@@ -237,7 +255,7 @@ impl<'a> BaseFunction<'a> for ImageToImage<'a> {
         &mut self.common
     }
     fn generate(&self) -> Result<(), WasmedgeSdErrno> {
-        if self.common.prompt.is_empty() {
+        if self.common.prompt.is_empty(){
```
Comment (hydai) [inline rust/src/lib.rs]: We should have a space here. Would you like to run cargo-fmt to fix the linter issue?

**Review by hydai — CHANGES_REQUESTED** [review]
You also put a wasm file here. Is this expected?

**Comment by grorge123** [comment]
> You also put a wasm file here. Is this expected?

I saw the WASI-NN example all have the wasm file. Is it unnecessary?

File: rust/src/lib.rs (hunk 1 above)
Comment (grorge123) [reply rust/src/lib.rs]: I have fixed it; please check. Thx.

**Comment by hydai** [comment]
> I saw the WASI-NN example all have the wasm file. Is it unnecessary?

Unnecessary. The reason is that we previously put these files so users don't need to compile them by themselves. However, it would be replaced by the released assets.

**Comment by hydai** [comment]
We are updating the function signature, so the `cargo.toml` version should be updated too.

**Review by hydai — CHANGES_REQUESTED** [review]
Let's remove the wasm file and bump the version. The remaining things are fine.

**Review by hydai — APPROVED** [review]

**Comment by hydai** [comment]
Merged and published 0.2.0 to the crate.io

**Pull request merged**